RPG Wizardry/Assets/Scripts/Sorcery/SpellPageManager.cs

```diff
@@ -82,7 +82,7 @@ internal void UnlockSpell()
         /// </summary>
         private void OnEnable()
         {
-            uint? dust = PlayerManager.Instance.Inventory?.Dust;
+            uint? dust = PlayerManager.Instance?.Inventory?.Dust;
```

Comment:
.Instance can never be null, as it uses Lazy Instantiation [https://docs.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization](url)